megamek/src/megamek/server/GameManager.java

```diff
@@ -499,7 +500,21 @@ public void checkForObservers() {
             p.setObserver((getGame().getEntitiesOwnedBy(p) < 1) && !getGame().getPhase().isLounge());
         }
     }
-    
+
+    //New SingleBlind - I dont think this is used.
+    /**
```

Comment:
```suggestion
    /**
```

Reply:
Thank you.  It will not let me accept or commit this one for some reason.  It says to refresh and try again, and just repeats when I do so.